onnxscript/type_annotation.py

```diff
@@ -28,31 +28,91 @@
 _LIST_CONSTRUCTORS = frozenset([list, typing.List, typing.Sequence, collections.abc.Sequence])
 
 
-def pytype_to_attrtype(pytype: type) -> typing.Optional[onnx.AttributeProto.AttributeType]:
+def remove_annotation(typeinfo):
+    """Remove Annotated wrapper if present, otherwise return typeinfo as is."""
+    if hasattr(typing, "Annotated"):
+        # Present in Python 3.9+
+        if get_origin(typeinfo) is typing.Annotated:
+            return get_args(typeinfo)[0]
+    return typeinfo
+
+
+def is_primitive_attr_type(typeinfo) -> bool:
```

Comment:
nit: annotations for params

Reply:
Would be nice. But seems non-trivial (to say anything other than `Any`).

Reply:
I think Any can be better than not annotated.